src/main/java/duke/DataErrorException.java

```diff
@@ -0,0 +1,6 @@
+package duke;
+
+public class DataErrorException extends DukeException{
+    // nothing to be done
+    // todo might not be used, can remove
```

Comment:
Typically you'd want to remove zombie code in your PR i.e. code that you're not going to use